android/fastlane/envfiles/wger.properties

```diff
@@ -0,0 +1,3 @@
+```properties
+WGER_API_KEY=77a270641bb5654c517e91f277bf06623c141318
```

Comment:
can you remove the file?

Reply:
Yeah no problem, sorry for that noobie mistake.

Reply:
Don't worry man